README.md

```diff
@@ -8,7 +8,9 @@ the source for the generation of the Flattened Image Tree Specification using
 Sphinx and LaTeX.
 
 ## Mailing list: flat-image-tree@lists.osfw.foundation ##
-   * Use this mailing list for submitting patches, questions and general discussion
+   * Use this mailing list for submitting questions and general discussion
+   * Patches should be submitted via a pull-request at
+     https://github.com/open-source-firmware/flat-image-tree/pulls
```

Comment:
suggestion (non-blocking): deduplicate info

The Contributions section already explains how it's done (plus some more info), so maybe avoid to repeat it here so we don't get contradicting info.

You could instead use a link something like

```md
For patches, please read the [Contributions section](#contributions)
```